modules/volunteers/sitesupport/volunteersEntry.js

```diff
@@ -0,0 +1,292 @@
+/*
+ * Javacript for the Volunteer Hour Entry
+ */
+
+/* jshint esversion: 6 */
+/* globals apiRequest, Vue, userLookup */
+/* exported root */
+
+var volApp = Vue.createApp({
+  created() {
+    console.log('load volunteer hours pane');
+    this.setEndDate();
+    apiRequest('GET', '/department','max_results=all')
```

Comment:
Here, again, if order matters, I'd consider making it a then-chain. But only if order matters.

Reply:
Again I tried to make sure it didnt.